src/kit.ts

```diff
@@ -552,7 +554,7 @@ async function collectDevBatVars(devbat: string, args: string[], major_version:
   } catch (error) { log.error(error); }
 
   if (!env || env === '') {
-    console.log(`Error running ${devbat} ${args.join(' ')} with:`, output);
+    console.log(`Error running:\n${batContent}\n ${args.join(' ')} with:`, output);
     return;
```

Comment:
2 more messages to localize

Reply:
`console.log` output does not need to be localized.

Reply:
Convert to log.error, easier for user to feedback.